calendar-bundle/src/Resources/contao/modules/ModuleCalendar.php

```diff
@@ -219,9 +220,23 @@ protected function compileWeeks()
 			$intFirstDayOffset += 7;
 		}
 
+		// Handle featured events
+		if ($this->cal_featured == 'featured')
+		{
+			$blnFeatured = true;
+		}
+		elseif ($this->cal_featured == 'unfeatured')
+		{
+			$blnFeatured = false;
+		}
+		else
+		{
+			$blnFeatured = null;
+		}
```

Comment:
```suggestion
		// Handle featured events
		$blnFeatured = null;

		if ($this->cal_featured == 'featured')
		{
			$blnFeatured = true;
		}
		elseif ($this->cal_featured == 'unfeatured')
		{
			$blnFeatured = false;
		}
```

Wäre um 2 Zeilen kürzer :)

Reply:
Fixed in d698402.